lib/datadog/appsec/metrics/telemetry_exporter.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Datadog
+  module AppSec
+    module Metrics
+      # A class responsible for exporting WAF request metrics via Telemetry.
+      module TelemetryExporter
+        module_function
+
+        def export_waf_request_metrics(waf_metrics, context)
```

Comment:
I think a method prefix is enough to know that metrics is about the WAF

```suggestion
        def export_waf_request_metrics(metrics, context)
```

Reply:
fixed in [f4de7ed](https://github.com/DataDog/dd-trace-rb/pull/4839/commits/f4de7edb068f62f5413a2678b9197d6063015c30)